### PR TITLE
[ADD] fork_projects: Adding a script to fork project from org1 to org2

### PR DESCRIPTION
**Pull request by moylop260**

### CODE REVIEW

File: tools/fork_projects.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    gh_login = login()
+
+    org_to = gh_login.organization(organization_to)
+    all_repos = gh_login.iter_user_repos(organization_from)
+    for repo in all_repos:
+        forked = False
+        try:
+            repo.create_fork(org_to)
+            forked = True
+        except github3.models.GitHubError, github_error:
```
Comment (gurneyalex) [inline tools/fork_projects.py]: This syntax is deprecated. Please use `except ExceptionClass as var`

File: tools/fork_projects.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+            pass
+        except TypeError:
+            try:
+                #  If has TypeError then use string to fix it
+                repo.create_fork(organization_to)
+                forked = True
+            except github3.models.GitHubError, github_error:
```
Comment (gurneyalex) [inline tools/fork_projects.py]: same as above

**Comment by moylop260** [comment]
@gurneyalex 
Fix it

File: tools/fork_projects.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+                             .format(repo_name=repo.name,
+                                     github_error=github_error.message))
+
+
+def main():
+    '''
+    Method main to get args and call fork method
+    '''
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--org_to",
```
Comment (guewen) [inline tools/fork_projects.py]: Using `_` in args is not really common, could you use `org-to`, `org-from` or `to`, `from`?

File: tools/fork_projects.py [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+                        help="Organization origin to get all repositories."
+                        " Default: oca",
+                        default='oca')
+    args = parser.parse_args()
+
+    fork(args.org_from, args.org_to)
+
+
+if __name__ == '__main__':
+    main()
```
Comment (guewen) [inline tools/fork_projects.py]: Can you add the script in a `setup.py` entry point?

**Comment by pedrobaeza** [comment]
Is this going to be finished?

**Comment by moylop260** [comment]
@pedrobaeza I will work in this PR in 3 weeks

**Comment by pedrobaeza** [comment]
No answer in a while, so closing. Feel free to reopen if still valid.

**Pull request closed without merging**